src/DeckrTable.js

```diff
@@ -93,6 +99,8 @@ export class DeckrTable extends Phaser.Game {
         const spinSpeed = Math.sqrt(((avx+bvx)**2) + ((avy+bvy)**2))
         chipA.setAngularVelocity(1 * (spinCoeff) * spinSpeed * -1)
         chipB.setAngularVelocity(1 * (spinCoeff) * spinSpeed)
+        // socket.emit('sendChip', { chip: chipA, room: gameState.room, otherPlayerDragging: false })
```

Comment:
I don't have strong feelings about this and I'm sure I've broken the rules myself, but in prior code reviews, Eliot seemed to think leaving commented out code was bad form.